project/Dependencies.scala

```diff
@@ -115,7 +115,7 @@ object Dependencies {
     excludeBouncyCastlePkix)
   val workbenchOpenTelemetryTest: ModuleID = "org.broadinstitute.dsde.workbench" %% "workbench-opentelemetry" % workbenchOpenTelemetryV % Test classifier "tests" excludeAll (excludeGuava)
 
-  val wsmClient: ModuleID = "bio.terra" % "workspace-manager-client" % "0.254.275-SNAPSHOT"
+  val wsmClient: ModuleID = "bio.terra" % "workspace-manager-client" % "0.254.445-SNAPSHOT"
```

Comment:
Huh - I see this was here already, but are we actually using the WSM client? Looks like everything is rolled by hand in `WsmDAO` ;-)